### ISSUE TITLE
sending images via slack?

### ISSUE THREAD

**Issue by dcsan** [silentrob/superscript#51]
I had a look at the slack docs to try and send an image but doesnt immediately work.

```
      if (reply) {
        reply.icon_url = "http://laorquesta.mx/wp-content/uploads/2014/12/bikers-300x225.jpg";
        channel.send(reply);
      }
```

wondering if we can just add extra fields to the reply in this way.

chat.postMessage should be able to just have an icon_url added.
https://api.slack.com/methods/chat.postMessage

images are a bit different.
https://api.slack.com/docs/attachments

done any experimentation here?

**Comment by dcsan** [comment]
> For the API, include an attachments property, containing a JSON-encoded array of attachment hashes.

OK it looks like youre just sending a string back, which gets converted somewhere else.
i tried sending json and got an error:

```
msgpack: { text: 'dont hello me. wheres your ticket?',
  icon_url: 'http://laorquesta.mx/wp-content/uploads/2014/12/bikers-300x225.jpg',
  attachment: 
   { color: '#36a64f',
     title: 'drifter bot',
     title_link: 'http://superscriptjs.com/starter/quickstart' } }
Error:
{ code: 2, msg: 'message text is missing' }
```

**Comment by dcsan** [comment]
odd because youre getting channel from their API

    channel = slack.getChannelGroupOrDMByName(user.name);
...
    channel.send(msgpack);

so that should accept more than a string.

**Comment by silentrob** [comment]
This is now landed on 0.5.0
